brave/brave-core
# feat(wallet): Plural String Handler in Panel UI

## Description 
Added `PluralStringHandler` in Panel WebUI


Resolves <https://github.com/brave/brave-browser/issues/23993>

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

### Testing Ethereum
1. Create a new wallet with 2 `Ethereum` accounts
2. Go to https://app.uniswap.org/#/swap
3. Open the `Wallet` panel and click the `Connect` button
4. It should say `0 accounts connected`
5. Click `connect` for your first account, it should now say `1 account connected`
6. Click `connect` for your second account, it should now say `2 accounts connected`

https://user-images.githubusercontent.com/40611140/179046878-3c394922-2ef1-497a-a8a8-43ff856bcc54.mov

### Testing Solana
1. Create a new wallet with 2 `Solana` accounts
2. Go to https://raydium.io/swap/
3. Connect to the dapp with your first account
4. Open the `Wallet` panel and click the `Connected` button
5. It should say `1 account trusted`
6. Click `trust` for your second account, it should now say `2 accounts trusted`
7. Click `revoke` for both accounts, it should now say `0 accounts trusted`

https://user-images.githubusercontent.com/40611140/179047802-3b33fd4c-d760-4b86-af70-7af68a5869a3.mov
